cmd/utask/root.go

```diff
@@ -129,6 +136,19 @@ var rootCmd = &cobra.Command{
 			}
 		}
 
+		sFiles, err := ioutil.ReadDir(utask.FScriptsFolder)
+		if err != nil {
+			log.Warnf("Ignoring scripts directory %s: %s", utask.FScriptsFolder, err)
+		}
+
+		if len(sFiles) > 0 {
+			for _, f := range sFiles {
+				if f.Mode()&0010 == 0 && f.Name()[0] != '.' {
```

Comment:
should we check the type? ie, regular file or symlink, to avoid folders, named pipes, etc?

Reply:
also, 010 seems weird

Reply:
We don't do that. I think it's the responsibility of the instance admin.
`010` match with rights of users belonging to the group. You think owners is needed too?

Reply:
@fsamin you make a good point, but still if something is wrong with my utask packaging, I'd rather know right away (and maybe have my orchestrator not go through with the update because it panics at boot), than find out a part of my workflows are not working later.

Reply:
IMO utask should check script files when it is parsing the templates... but not check ALL the files in this directory
BTW if the script is invalid (eg. syntax error) the workflows won't work, then we have to handle this error a nice way within the web ui

Reply:
@fsamin maybe the script plugin's **ValidConfig()** could check file existence and +x ? That way only the used script files would be checked. And afaik **ValidConfig()** would be called in that code path, because it is parsed + validated before being updated? cc @nomonamo confirm ?

Reply:
Validating script files that way (via **ValidConfig()**) would prevent the use of templating for the _file_ field, as a byproduct (because _{{.input.foobar}}_ is probably not an existing file).
This might not be a drawback, re: the discussion started below by @fsamin